aim/ext/transport/client.py

```diff
@@ -76,15 +77,20 @@ def _create_remote_channels(self):
 
         ssl_certfile = os.getenv(AIM_CLIENT_SSL_CERTIFICATES_FILE)
         msg_max_size = int(os.getenv(AIM_RT_MAX_MESSAGE_SIZE, AIM_RT_DEFAULT_MAX_MESSAGE_SIZE))
+        bearer_token = os.getenv(AIM_RT_BEARER_TOKEN)
         options = [
             ('grpc.max_send_message_length', msg_max_size),
             ('grpc.max_receive_message_length', msg_max_size)
         ]
 
         # open a channel with router
         if ssl_certfile:
+            self._call_credentials = grpc.access_token_call_credentials(bearer_token)
```

Comment:
Let's add an explicit check that bearer token exists, and if so, then add the credentials:
```
if bearer_token:
    self._call_credentials = grpc.access_token_call_credentials(bearer_token)
    root_certificates = grpc.composite_channel_credentials(
         root_certificates, self._call_credentials
     )
```